lips/lip-6.mdx

```diff
@@ -10,27 +10,26 @@ created: 07/30/2020
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # Summary
-This LIP describes the conceptual governance model of the Libra blockchain, in particular, the on-chain Validator Set configuration.
-Validator Set is part of a global on-chain configuration stored as part of the blockchain state, it additionally includes the list of registered currencies, publishing policies and Libra version.
-Validator Set defines a collection of entities, called Validator Nodes, that coordinate and reach consensus on the state of the blockchain by processing transactions, they ensure correct updates to the state even if some Validator Nodes (fewer than a third) fail in arbitrary ways. The LibraRoot account governed by the Libra Payment Network and the Libra Association defines the composition of the validator set. LibraRoot may add or remove members over time.
+This LIP describes the organization of Libra's proof of authority consensus model by which the Libra blockchain agrees upon ledger state. In a proof of authority consensus model, well-known participants leverage cryptographical digital signatures to agree upon a set of transactions and their output to advance the blockchain's state. The set of potential entities participating in consensus are known as Validators, while the active participants are known as the Validator Set. The Validator Set is part of a global on-chain configuration stored as part of the ledger state. Other examples of on-chain configuration include the list of registered currencies, publishing policies and Libra version. LibraRoot is the sole entity capable of adding or removing Validators and specifying the current Validator Set.
 
 ## LibraRoot account management
-The Libra Payment Network (“LPN”) is a blockchain-backed payment platform and an infrastructure for stablecoins. Libra Networks (“Libra Networks” or “LN”) is a whole subsidiary of the Libra Association (the “Association” or “LA”) which is in the process of applying for an LPN payment system license from the Swiss Financial Market Supervisory Authority (“FINMA”). Once granted, LN will be responsible for ensuring that LPN operates in compliance with the payment system license from FINMA (the "FINMA License").
+The Libra Payment Network ("LPN") and the Libra Association ("LA") manage the LibraRoot account. The LPN is a blockchain-backed payment platform and an infrastructure for stablecoins. Libra Networks (“Libra Networks” or “LN”) is a whole subsidiary of the LA which is in the process of applying for an LPN payment system license from the Swiss Financial Market Supervisory Authority (“FINMA”). Once granted, LN will be responsible for ensuring that LPN operates in compliance with the payment system license from FINMA (the "FINMA License").
```

Comment:
nit: Maybe change to: "Libra Networks (or "LN") is a subsidiary of the LA. The LA is currently in the process of applying for a LPN payment system license..."